bug-finding.ipynb

```diff
@@ -185,6 +222,81 @@
     "# Bug 1 analysis and response"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I believe this is because lemmata is not properly defined. That is, I think the code should be: \n",
+    "lemmata = [t.lemma_ for t in df['nlp_docs].explode() \n",
+    "\n",
+    "I also changed the / len (lemmata) to / len(df['tokens].explode()), since I believe we need to divide by the number of tokens--not the number of lemmata--in the corpus in order to get the expected collocation frequency. \n",
```

Comment:
Think about what the difference is between `[t.lemma_ for t in df['nlp_docs'].explode()]` versus `df['tokens'].explode()` -- will they actually be different lengths?